draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4059,7 +3762,7 @@ role. The content of the attribute is a 64-bit unsigned integer in
 network byte order, which contains a random number. The number is used
 for solving role conflicts, when it is referred to as the tie-breaker
 value. An ICE agent MUST use the same number for all Binding requests,
-for all streams, within an ICE session. The ICE agent MAY change
+for all streams, within an ICE session. The ICE agent MUST NOT change
```

Comment:
It seems that if some people implement changing this, then a new thing that says MUST NOT is going to fail when talking to them